### ISSUE TITLE
Why has fsync been disabled?

### ISSUE THREAD

**Issue by Atemu** [lutris/wine#83]
6.1 release notes say:

> Fsync is removed (due to issues in

...?

Fsync works just fine on my machine with 5.7.

**Comment by liberodark** [comment]
In 6.10 i think is not disable no information about that.

**Comment by SveSop** [comment]
Cant see any mention of using fsync patches [https://github.com/lutris/wine/blob/lutris-6.10-2/wine-tkg-config.txt](https://github.com/lutris/wine/blob/lutris-6.10-2/wine-tkg-config.txt)

Maybe disabled due to issues, and forgot enable it again? :) No issues with fsync or futex2 on TKG or other custom wine versions that i know of, so i am not sure.

**Comment by liberodark** [comment]
Same on my side have no issue on fsync probably need to wait for re enable on lutris.

**Comment by tannisroot** [comment]
The patch seems to cause issues in Guild Wars 2 for some users even when fsync is disabled. Given that Fsync requires a custom kernel which not many are going to install / compile, i decided to remove it for now, since the users who have the knowledge base to install a new kernel and actually use it should be able to use wine-tkg to compile wine with fsync enabled.

**Comment by Atemu** [comment]
The GW2 issue (which I was also hit by; it's my main game on Linux) was an upstream bug that should have been fixed in 6.5 AFAIK.

 > Given that Fsync requires a custom kernel which not many are going to install / compile

You'd be surprised how many people use fsync capable kernels. Arch has the zen-kernel in their core repos, there's lqx for all the Debianish distros and I've added both to Nixpkgs/NixOS.

Also, since this only affected one game's launcher, a -gw2 version similar to the -lol releases etc. might have been more appropriate.

> i decided to remove it for now, since the users who have the knowledge base to install a new kernel and actually use it should be able to use wine-tkg to compile wine with fsync enabled.

Able to, yes. But installing a custom kernel (especially when binary caches are available; in NixOS it's a one-line change) is a lot easier than cooking up a custom WINE and getting Lutris to use that.

The official Lutris binaries are simply more convenient.
